tests/installation/bootloader_uefi.pm

```diff
@@ -117,10 +117,8 @@ sub run() {
     }
 
     type_string " \\\n";    # changed the line before typing video params
-                            # https://wiki.archlinux.org/index.php/Kernel_Mode_Setting#Forcing_modes_and_EDID
```

Comment:
I removed it as it's documented inside bootmenu_default_params - this is just unrelated duplicity here - I can open separate PR if needed